lib/core/utils/preload-cssom.js

```diff
@@ -27,25 +26,26 @@ axe.utils.preloadCssom = function preloadCssom({
 	 */
 	const rootNodes = getAllRootNodesInTree(treeRoot);
 
-	const q = axe.utils.queue();
+	const promises = [];
 
 	if (!rootNodes.length) {
-		return q;
+		return Promise.all(promises);
 	}
 
 	const dynamicDoc = document.implementation.createHTMLDocument();
 	const convertDataToStylesheet = getStyleSheetFactory(dynamicDoc);
 
-	q.defer((resolve, reject) => {
+	const p = new Promise((resolve, reject) => {
```

Comment:
Rather than wrapping this in a Promise, we should just use the Promise returned by `getCssomForAllRootNodes` directly.